controllers/territory_controller/territory_controller.py

```diff
@@ -69,13 +71,18 @@ def log_territory_claim(
         claim_time: float,
     ) -> None:
         self._log.append((station_code, claimed_by, claim_time))
+        self._log_is_dirty = True
         print(f"{station_code} CLAIMED BY {claimed_by} AT {claim_time}s")  # noqa:T001
         self._station_statuses[station_code] = claimed_by
 
     def record_captures(self) -> None:
         if not self._record_arena_actions:
             return
 
+        if self._log_is_dirty:
```

Comment:
Should this check be inverted? The log is dirty so we want to write it to disk rather than exiting out early.
```suggestion
        if not self._log_is_dirty:
```

Reply:
Very true